app/handler/trades.js

```diff
@@ -841,6 +842,9 @@ exports.offerChanged = function (offer, oldState) {
 
         if (offer.state === TradeOfferManager.ETradeOfferState.Accepted) {
             admin.message('Trade #' + offer.id + ' with ' + offer.partner.getSteamID64() + ' is accepted. Summary:\n' + offer.summarize());
+            if (process.env.DISCORD_WEBHOOK_URL) {
```

Comment:
You used upper case 'L' here in `DISCORD_WEBHOOK_URL` but lower case everywhere else.

Reply:
Lol idk how that happened, should be uppercase everywhere